website/src/pages/index.tsx

```diff
@@ -218,30 +199,29 @@ const Hero = ({ language }) => (
       </div>
 
       <DummyMiniRepl />
-      <MiniRepl language={language} />
+      <MiniRepl />
     </div>
   </div>
 );
 
-const Index = ({ language }) => {
-  const { siteConfig } = useDocusaurusContext();
+const Index = () => {
   return (
-    <Layout title={siteConfig.title} description={siteConfig.tagline}>
```

Comment:
This is a fix.

docusaurus will automatically apply them.
If we specify it manually, it will be repeated.
Below is the current main behavior.
![image](https://github.com/user-attachments/assets/f88dc331-49da-4c20-ba06-68990227bb74)